.gitignore

```diff
@@ -6,3 +6,7 @@ pom.xml.next
 release.properties
 dependency-reduced-pom.xml
 buildNumber.properties
+.classpath
+.project
+.settings
+.gitignore
```

Comment:
@raidenovski The `.gitignore` file itself probably shouldn't be ignored?

Reply:
Will remove it from `.gitignore`